Shared/Utils/EventUtils.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  EventUtils.swift
+//  InternxtDesktop
+//
+//  Created by Patricio Tovar on 21/5/24.
+//
+
+import Foundation
+
+struct EventsUtils {
```

Comment:
Wouldn't be easier to just do this?:

`Analytics.shared.track(event: SuccessBackupEvent(foldersToBackup: foldersToBackup))`

So we use always the same pattern when sending analytics events